docs/src/configuration/services/varnish.md

```diff
@@ -14,6 +14,8 @@ However, it is possible to configure a Varnish instance as part of an applicatio
 |----------------------------------|---------------|---------------|
 |  {{< image-versions image="varnish" status="supported" environment="grid" >}} | {{< image-versions image="varnish" status="supported" environment="dedicated" >}} | {{< image-versions image="varnish" status="supported" environment="dedicated-gen-3" >}} |
```

Comment:
For many services, the array is empty for some columns. Do we want to display `None available`?

Reply:
It should do so already :thinking:

Reply:
Even after a force-refresh, I get:
<img width="1048" alt="Screenshot 2022-02-18 at 15 55 50" src="https://user-images.githubusercontent.com/16886290/154706584-30872be9-eae9-4e34-b74f-2278e6abd660.png">